api/app/signals/settings/base.py

```diff
@@ -17,15 +17,11 @@
 # Django settings
 SECRET_KEY = os.getenv('SECRET_KEY')
 DEBUG = False
-ALLOWED_HOSTS = [
-    'api.data.amsterdam.nl',
-    'acc.api.data.amsterdam.nl',
-
-    # Currently this is needed because the deployment process checks the health endpoint with a
-    # request to localhost:port.
-    'localhost',
-    '127.0.0.1',
-]
+
+# localhost and 127.0.0.1 are allowed because the deployment process checks the health endpoint with a
+# request to localhost:port
+ALLOWED_HOSTS = os.getenv('ALLOWED_HOSTS', 'api.data.amsterdam.nl,acc.api.data.amsterdam.nl,localhost,127.0.0.1').split(',')
```

Comment:
@bartjkdp I just run the test suite locally (because our infra does not allow automated runs from the test suite for forks) and I just had 1 error, Flake8: `signals/settings/base.py:23:121: E501 line too long (124 > 120 characters)
ERROR: InvocationError for command /usr/local/bin/flake8 signals tests (exited with code 1)`

Reply:
This is the only problem I see as well.

Reply:
Thanks, I will fix that.

Reply:
Fixed!